.github/workflows/wheels.yml

```
@@ -38,18 +38,19 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.7", "3.8", "3.9", "pypy-3.7"]
 
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     - uses: pypa/cibuildwheel@2.7.0
       env:
         CIBW_TEST_EXTRAS: test
         CIBW_TEST_COMMAND: "pytest {project}/tests"
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
         CIBW_MANYLINUX_I686_IMAGE: manylinux1
-        CIBW_SKIP: pp* cp27-*-win* cp36-*
+        CIBW_SKIP: pp* cp27-* cp36-*
```

```suggestion
        CIBW_SKIP: pp* cp36-*
```

Looks like we might support PyPy too? Anyway, this will fix the warning.

---

Oh, it's separate below.

---

Thanks.